### PR TITLE
Download and View Icons Clearer for Students

### PR DESCRIPTION
**Pull request by thef1stofhername**
I have made the download and view buttons larger and blue with a description of what the icon does (Download or View) with the last Download button change to occur whenever pdf annotations get put in. (and yes I did have to google how to screencap on linux)
Screenshot from 2019-06-14 16-58-23

### CODE REVIEW

File: site/app/controllers/grading/ElectronicGraderController.php [hunk 1]
```diff
@@ -335,6 +336,7 @@ public function showStatus() {
             $component_averages = array();
             $autograded_average = null;
             $overall_average = null;
+            $overall_scores = null;
```
Comment (xprilion) [inline site/app/controllers/grading/ElectronicGraderController.php]: This, and the above 3 lines are not needed

File: site/app/libraries/database/DatabaseQueries.php [hunk 2]
```diff
@@ -957,6 +957,57 @@ public function getAverageAutogradedScores($g_id, $section_key, $is_team) {
...
+                ON gd.g_id=auto.g_id AND gd_{$user_or_team_id}=auto.{$user_or_team_id}
+                INNER JOIN {$users_or_teams} AS {$u_or_t} ON {$u_or_t}.{$user_or_team_id} = auto.{$user_or_team_id}
+                WHERE gc.g_id=? AND {$u_or_t}.{$section_key} IS NOT NULL
+              )AS parts_of_comp
+            )AS comp
+            GROUP BY gd_id, autograding
+          )g 
+      ", array($g_id));
+        if(count($this->course_db->rows()) == 0){
+          echo("why");
```
Comment (xprilion) [inline site/app/libraries/database/DatabaseQueries.php]: Is this a part of the program or an echo checkpoint?

Comment (thef1stofhername) [reply site/app/libraries/database/DatabaseQueries.php]: I'll have to redo the PR on Monday. I just noticed it pulled in some files that I didn't change in there. Please ignore for now

Comment (xprilion) [reply site/app/libraries/database/DatabaseQueries.php]: Sure, was wondering the same when going through! Thought it was out of my understanding to see a `sql.txt` there :laughing:

Comment (thef1stofhername) [reply site/app/libraries/database/DatabaseQueries.php]: Lol I was trying to push like 3 branches in sequence... lesson learned :p

**Comment by bmcutler** [comment]
Whoops, looks like you deleted the files rather than restore the files to the original.
Also, we discussed reworking the page layout to eliminate some unnecessary content and more clearly label the annotated files vs original files.

**Pull request closed without merging**